docs/running/amazon.rst

```diff
@@ -0,0 +1,211 @@
+.. _runningAWS:
+
+Running in AWS
+==============
+
+Prepare your AWS environment
+----------------------------
+1. If necessary, create and activate an `AWS account`_
+
+.. _AWS account: https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/ 
+
+
+2. Create a `key pair`_ in the ``us-west-2a`` availability zone. 
+
+.. _key pair: http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html#having-ec2-create-your-key-pair 
+.. important::
+
+   This will automatically download a PEM file from your Web browser to your local machine.  Put this file in your home directory or somewhere memorable.
+
+3. Add the AWS private key to the authentication agent.  
+::
+
+   $ (venv) ssh-add <path-to-aws-pem-file>
+
+4. Create an `AWS access key`_
+
+.. _AWS access key: http://docs.aws.amazon.com/general/latest/gr/managing-aws-access-keys.html 
+
+5. Export AWS access key environment variables in shell.  
+::
+
+   $ (venv) export AWS_ACCESS_KEY_ID=<access-key-string>
+   $ (venv) export AWS_SECRET_ACCESS_KEY=<secret-access-key-string>
+
+.. note::
+
+   Instead of typing the above ``ssh-add`` and ``export`` commands every time you wish to access AWS, you could instead put them in your shell initialization file (e.g. ``~/.bash_profile``).
+
+
+Launch a Toil workflow in AWS
+-----------------------------
+The user can run the same ``HelloWorld.py`` script on a distributed cluster just by modifying the run command. Since our cluster is distributed, we'll use the ``aws`` job store which uses a combination of one S3 bucket and a couple of SimpleDB domains.  This allows all nodes in the cluster access to the job store which would not be possible if we were to use the ``file`` job store with a locally mounted file system on the leader.
+
+1. Launch a mesos cluster in AWS.
+::
+
+   $ (venv) toil launch-cluster <cluster-name> \
+   --keyPairName <AWS-key-pair-name> \
+   --nodeType t2.micro \
+   --zone us-west-2a 
+
+2. Copy ``HelloWorld.py`` to the leader node.  
+:: 
+
+$ (venv) toil rsync-cluster <cluster-name> HelloWorld.py :/tmp
+
+3. Run the Toil script in the cluster.  
+::
+
+   $ python HelloWorld.py \
+   --batchSystem mesos \
```

Comment:
I don't think using the mesos batchSystem is necessary here since this is a static cluster and you're not using autoscaling. If you intended to use autoscaling then the `--provisioner aws` argument (and nodeType) should be added.